.github/workflows/perf.yml

```diff
@@ -147,28 +148,26 @@ jobs:
         with:
           path: artifacts
           
-      - name: Produce performance model table
-        run: python e2etest/get-performance-model-table.py
-      
-      - name: Create a new branch
-        uses: peterjgrainger/action-create-branch@v2.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          branch: "perf-test-${{ github.run_id }}"
-          
-      - name: Commit to a branch
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: "Update benchmarking"
-          branch: "perf-test-${{ github.run_id }}"
-          file_pattern: docs/performance_model.md
-      
-      - name: pull-request
-        uses: repo-sync/pull-request@v2
+      - name: Produce performance report
+        run: python e2etest/get-performance-model-table.py ${{ needs.get-testing-version.outputs.testing_version }}
```

Comment:
IMO, we should use flags instead of passing environment variables since we could have more flags for future use and it would make more clarification when catching these flags in the python code.

Reply:
Added a -v/--version flag.